src/bundler.ts

```diff
@@ -155,7 +157,12 @@ const createFinalBundles = async (bundles: Bundle[], distDirectory: string, buil
   await Promise.all(renamingOps)
 }
 
-const getBasePath = (sourceDirectories: string[]) => {
+const getBasePath = (sourceDirectories: string[], inputBasePath?: string) => {
+  // If there's a specific base path supplied, that takes precedence.
+  if (inputBasePath !== undefined) {
```

Comment:
Should we do 
```suggestion
  if (inputBasePath != undefined) {
```
in case it is null?

Reply:
The types are saying that it's either `string` or `undefined`, so `null` feels like a case that we shouldn't worry about?

Reply:
True.